webapp/portlet/pom.xml

```diff
@@ -123,6 +123,8 @@
                 <include>portlet/SuggestionsPeopleAndSpace/Style.less</include>
                 <include>portlet/TopBarMenu/Style.less</include>
                 <include>portlet/WhoIsOnLine/Style.less</include>
+                <include>portlet/VerticalMenu/Style.less</include>
+                <include>common/siteDetails/Style.less</include>
```

Comment:
It seems that those Style files are useless, please see comments below.

```suggestion
```

Reply:
As [mentioned here](https://github.com/Meeds-io/social/pull/3229#discussion_r1404536475), we will keep this for now to not block the merge. But please, we doesn't need CSS classes anymore since we use Material Design classes, Vuetify classes and specific CSS Class helpers.